### PR TITLE
interventions: document holding all tasks, triggering and releasing

### PR DESCRIPTION
**Pull request by oliver-sanders**
Document a common use case:
* Hold all tasks
* Trigger some tasks
* Release all tasks

Currently the solution is to use the "hold after cycle point" feature.

Note, when https://github.com/cylc/cylc-flow/issues/5827 is done, we will be able to simplify this to:

```
$ cylc hold 'workflow//*'
$ cylc trigger ...
$ cylc release 'workflow//*'
```

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.

### CODE REVIEW

File: src/user-guide/interventions/index.rst [hunk 1]
```diff
@@ -439,3 +439,52 @@ Terminate a Chain of Automatic Retries
...
+         $ # release the "hold point" to allow the workflow to continue
+         $ cylc release --all <workflow>
+
+.. note::
+
+   The difference between the workflow "paused" state and the task "held" state:
+
+   Workflow Pause
+      When a workflow is :term:`paused <pause>`, no new jobs will be submitted.
+      This gives you an oportunity to make changes to the workflow, however, if
```
Comment (markgrahamdawson) [inline src/user-guide/interventions/index.rst:485]:
```suggestion
      This gives you an opportunity to make changes to the workflow, however, if
```

File: src/user-guide/interventions/index.rst [hunk 2]
```diff
@@ -439,3 +439,52 @@ Terminate a Chain of Automatic Retries
...
+   * Trigger the task(s) you want to run.
+   * When you're done triggering, release the held tasks to allow the workflow
+     to continue.
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: set-and-release-hold-point.gif
```
Comment (MetRonnie) [inline src/user-guide/interventions/index.rst:462]: This is quite a lengthy GIF which could be halfway through playback when the user starts trying to follow along with it. Is it possible to embed a video instead?

Comment (oliver-sanders) [reply src/user-guide/interventions/index.rst:462]: Dunno.

Comment (oliver-sanders) [reply src/user-guide/interventions/index.rst:462]: Run the GIF through once, then stop and provide a button to play from the start is another option.

**Comment by oliver-sanders** [comment]
(merging with two approvals)

**Pull request merged**